src/Microsoft.DotNet.ProjectModel/RuntimeConfig/RuntimeConfig.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.IO;
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    public class RuntimeConfig
```

Comment:
cc @anurse I think this abstraction will need to move and be less hacky, but for the moment does this appear correct?